overcooked_ai_py/agents/agent.py

```diff
@@ -205,25 +209,29 @@ class FixedPlanAgent(Agent):
     # NOTE: Assumes that calls to action are sequential (agent has history)
     """
 
-    def __init__(self, plan):
+    def __init__(self, plan, return_action_probs=True):
         self.plan = plan
         self.i = 0
+        self.return_action_probs = return_action_probs
     
     def action(self, state):
         if self.i >= len(self.plan):
             return Action.STAY, {}
         curr_action = self.plan[self.i]
         self.i += 1
-        return curr_action, {}
-    
+        if self.return_action_probs:
+            return curr_action, {}
```

Comment:
Always return empty dictionary, infrastructure should always expect an (action, info_dictionary) pair as return to the action method, even if such dictionary is empty.